wbooks-api/app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Api
+  module V1
+    class RentsController < ApiController
+      def index
+        rents = Rent.all
+        render_paginated rents, each_serializer: RentSerializer
+      end
+
+      def show
+        rent = Rent.find(params[:id])
+        render json: rent, serializer: RentSerializer
+      end
+
+      def create
+        rent = Rent.new(rent_params)
+        if rent.save
+          head :ok
```

Comment:
We usually return the created object in POST requests

Reply:
I agree, and use https://quickrails.com/ruby-one-line-if-else-statement/

By the way, I agree with this https://en.wikipedia.org/wiki/Fail-fast

Return Ok at the end.